ang/civicase/DashboardTab.html

```diff
@@ -1,5 +1,194 @@
 <div class="civicase__dashboard__tab">
-  <div class="civicase__dashboard__tab__top"></div>
+  <div class="civicase__dashboard__tab__top">
+    <div class="panel panel-secondary civicase__dashboard__overview">
+      <div class="panel-body">
+        <div class="civicase__dashboard__overview-container">
+          <div class="civicase__dashboard__overview__title">
+            <div class="civicase__dashboard__overview__title-field">
+              Overview
+            </div>
+            <div class="civicase__dashboard__overview__title-field">
```

Comment:
`.civicase__case-overview__flow__status`

Reply:
Made it `civicase__case-overview__flow-status`